### DISCUSSION TITLE
[FR] How to versioning local mockups

### DISCUSSION THREAD

**Discussion by Nevu** [ingrammicro/puzzle-publisher#55] [General]
Hi,

do you have a how to, how-to-versioning a file? Is it also possible with local files?
Would be a nice feature I'd like ti use!

Regards

**Comment by MaxBazarov** [comment]
Automatic versioning is the part of "Publish HTML" feature. But currently it works only for SFTP.

But I can integrate a versioning into "Generate HTML" also. So, it will generate HTML into <YOUR FOLDER>/live and <YOUR FOLDER>/<NEW VERSION> folders. Will it be fine for you?

**Reply by Nevu** [reply]
Would be great to have the option to display the version also in local files, yes!
